__init__.py

```diff
@@ -687,9 +686,11 @@ def handle_stt_menu(self):
         for engine, configs in model.items():
             # For Display purposes, we want to show the engine name without the underscore or dash and capitalized all
             plugin_display_name = engine.replace("_", " ").replace("-", " ").title()
-            supported_stt_engines.append({"plugin_name": plugin_display_name,
-                                          "offline": configs[0]["offline"],
-                                          "engine": engine})
+            for config in configs:
+                supported_stt_engines.append({"plugin_name": plugin_display_name,
+                                              "display_name": config.get("display_name", "unknown"),
```

Comment:
instead of "unknown" maybe this could default to something else, maybe use the lang as default name?

Reply:
fixed, leaving it empty for now